.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    branches:
+      # todo: change this branch name
+      - master
+
+jobs:
+  generate-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          # semantic-release requires >= 10.18
+          node-version: 12
+
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        env:
+          cache-name: node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ env.cache-name }}-
+
+      # deletes node_modules, install dependencies, does NOT update lockfile
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Semantic Release
```

Comment:
Does this step need the bot token?

Reply:
It does and I have no idea how I missed that! 😵